style.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  max-width: 480px;
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background: hsl(20, 3%, 23%);
+  width: 100%;
+  height: 48px;
+  margin: auto;
+}
+
+.hambuger {
+  border-radius: 8px;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  margin-right: 20px;
+}
+
+.logo {
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  color: white;
+  font-size: 18px;
+  font-family: sans-serif;
+  padding: 10px 12px;
+  margin-left: 20px;
+  text-decoration: none;
+}
+
+.hambuger .logo {
+  flex: 1;
+}
+
+.primary-text {
+  margin-bottom: 12px;
+}
+
+h1 {
+  color: hsl(25, 100%, 50%);
+  font-family: 'Crete Round', sans-serif;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+}
+
+.supporting-text h3 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  color: hsl(40, 100%, 94%);
+  margin-bottom: 12px;
+}
+
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: white;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  background: hsl(20, 6%, 10%) url(images/bgvector.svg) no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 956px;
+  padding: 0;
+  width: 100%;
+  margin: auto;
+}
+
+.hero {
+  padding: 120px 24px;
+}
+
+.social-icons {
+  display: flex;
+  justify-content: space-between;
```

Comment:
[Optional] Please kindly check the `justification` and `left margin` of your social media icons. They are not exact like those of the Figma design. Try to fix this as you continue working on your portfolio. It will give a beautiful user experience.

Your design 
![image](https://user-images.githubusercontent.com/62849872/143557640-3e023956-fbcd-4b1a-b25e-b1bf55ef4796.png)

Figma design 
![image](https://user-images.githubusercontent.com/62849872/143557740-e281c7f8-17f0-460d-9d9f-48ae9e76478f.png)